qiime2/sdk/usage.py

```diff
@@ -28,10 +50,24 @@ def __init__(self, *, plugin_id: str, action_id: str):
         self._plugin_manager = sdk.PluginManager()
 
     def __repr__(self):
-        return 'UsageAction(plugin_id=%r, action_id=%r)' %\
-            (self.plugin_id, self.action_id)
+        return 'UsageAction(plugin_id=%r, action_id=%r)' % \
+               (self.plugin_id, self.action_id)
+
+    def get_action(
+        self,
+    ) -> typing.Tuple[typing.Union['sdk.Method', 'sdk.Pipeline'],
```

Comment:
This is minor, and is filed under "stylistic concerns," but splitting `self` out onto its own line seems unnecessary to me and just adds noise to the PR diff. Changing the prevailing style of pre-existing code just seems like extra work. Take that comment with a grain of salt, though: I like to play diff-golf.

Reply:
I forgot to mention, but there are a ton of other examples of this sprinkled throughout this PR, I'm not too sure if that is really a necessary change to the module.